app/hotels/src/filter/FilterParametersType.js

```diff
@@ -17,3 +17,10 @@ export type OnChangeFilterParams = {
   hotelFacilities?: string[],
   minScore?: ?number,
 };
+
+export type ActiveFilters = {
+  isPriceFilterActive: boolean,
+  isStarsFilterActive: boolean,
+  isMinScoreActive: boolean,
+  isHotelFacilitiesActive: boolean,
+};
```

Comment:
Please use everywhere in this file exact types - should be possible. I noticed there is an error with `OnChangeFilterParams` type.